doc/source/changes.rst

```diff
@@ -2,6 +2,91 @@
 What's New in Workload Automation
 =================================
 
+***********
+Version 3.3
+***********
+
+New Features:
+==============
+
+Commands:
+---------
+    - Add ``report`` command to provide a summary of a run.
+
+Instruments:
+------------
+    - Add ``proc_stat`` instrument to monitor CPU load using data from ``/proc/stat``.
+
+Framework:
+----------
+    - Add support for simulating atomic writes to prevent race conditions when running current instances of WA.
+    - Add support file transfer for SSH connections via SFTP and falling back to using SCP implementation.
+    - Support detection of logcat buffer overflow and present a warning if this occurs.
+    - Allow skipping all remaining jobs if a job had exhausted all of its retires.
+    - Add polling mechanism for file transfers rather than relying on timeouts.
+    - Add `run_completed` reboot policy to enable rebooting a target after a run has been completed.
+
+
+Android Devices:
+----------------
+    - Enable configuration of whether to keep the device on while the device is plugged in.
```

Comment:
"whether to keep the device on" ? Is this referring to the screen?

Reply:
Thanks, that should say screen.